src/test/java/com/coveo/spillway/SpillwayTest.java

```diff
@@ -118,20 +121,17 @@ public void simpleLimit() {
     assertThat(spillway.tryCall(john)).isFalse(); // Third tryCall fails
   }
 
-  @Test
-  public void multipleLimitsWithOverlap() {
+  @Test(expected = SpillwayLimitWithSameNameException.class)
+  public void multipleLimitsWithOverlap() throws Exception {
```

Comment:
Like I said in this case just use two limits since we now check the limits before incrementing them.